src/ray/common/id.cc

```diff
@@ -20,7 +20,7 @@ extern "C" {
 
 namespace ray {
 
-uint64_t MurmurHash64A(const void *key, int len, unsigned int seed);
+uint64_t MurmurHash64B(const void *key, int len, uint64_t seed);
```

Comment:
The `int` should also be `size_t`.

Reply:
This code is copied from murmurhash, but I removed it in https://github.com/ray-project/ray/pull/6701 (we might need to bring it back thought.